docs/reference/api/widgets/detailedlist.rst

```diff
@@ -80,17 +75,17 @@ icon, title, and subtitle, respectively:
         ]
     )
 
-If the value provided by the accessor for title or subtitle is :any:`None`, or the
-accessor isn't defined, the value of ``missing_value`` provided when constructing the
-DetailedList will be used to populate the title and subtitle. Any other value will be
+If the value provided by the accessor for ``title`` or ``subtitle`` is :any:`None`, or
```

Comment:
I didn't quote these because we're just after the "you can customise the accessors" example, so I was trying to refer to "title" as an abstract idea, rather than ``title`` as a specific key. Not sure if that's a helpful or meaningful distinction, but raising it as a weakly held opinion in case you agree.

Reply:
Yes, that totally makes sense. Fixed.